src/sage/modules/free_module_integer.py

```diff
@@ -172,6 +172,19 @@ def IntegerLattice(basis, lll_reduce=True):
         [      0       0       0       0       0 1048576       0       0]
         [      0       0       0       0       0       0 1048576       0]
         [      0       0       0       0       0       0       0 1048576]
+
+
+    We construct a large lattice, this lattice will not be constructible 
+    if echelon form was also being computed::
```

Comment:
```suggestion

    We construct a large lattice, this lattice will not be constructible 
    if echelon form was also being computed::
```
Too many blanklines.